Kconfig

```diff
@@ -110,11 +116,7 @@ config SECURE_BOOT
 	  This uses the public custMpk.key and certificate by default. Do not
 	  use this key in production, it is for demonstration purposes only.
 
-	  The writable persistent /var and /home partitions are encrypted as
-	  LUKS2 partitions.
-
-	  The reason of coupling the data encryption with the secure boot is to
-	  have a proper integrity chain rooted in hardware (OTP, RPMB, etc.)
+	  Note: The persistent /var and /home partitions are not yet protected!
```

Comment:
Unrelated changes.

Reply:
There was a rebase issue, now it is aligned with master branch.